database/seeds/ResearchSeeder.php

```diff
@@ -0,0 +1,39 @@
+
+
+<?php
+
+use Illuminate\Database\Seeder;
+
+class ResearchSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     */
+    public function run()
+    {
+        DB::table('research')->insert([
```

Comment:
Is there any reason why we're not using the model directly?

Reply:
because were not using factories for our seeders. Liad finds this more readable

Reply:
daddy liad promises that we'll be good with renaming any names in the event we have to change anything in our migration. uWu